config/sync/filter.format.hearing_description.yml

```diff
@@ -0,0 +1,45 @@
+uuid: 15881684-2eac-4426-bae4-7e44d395c82a
+langcode: da
+status: true
+dependencies:
+  module:
+    - token_filter
+name: 'Hearing description'
+format: hearing_description
```

Comment:
Probably not worth a change, but generally i'm not sure our filters should be attached to specific fields or entities.

Reply:
The `[gis:minimap:?]` token makes sense only in the hearing description field, and therefore a new text format with the token filter has been added.